src/StorageSync/StorageSync/StorageSync.csproj

```diff
@@ -10,6 +10,13 @@
     <RootNamespace>$(LegacyAssemblyPrefix)$(PsModuleName)</RootNamespace>
   </PropertyGroup>
 
+  
+  <ItemGroup>
+    <PackageReference Include="Microsoft.Azure.Management.StorageSync" Version="2.8.0-preview" />
+    <PackageReference Include="Microsoft.Win32.Registry" Version="4.5.0" />
+    <PackageReference Include="System.Management" Version="4.5.0" />
```

Comment:
Note that you will need to include the assembly from the netcoreapp2.0 runtime here

Reply:
I dont understand this comment. Please rephrase or suggest change.

---
In reply to: [255703634](https://github.com/Azure/azure-powershell/pull/8465#discussion_r255703634) [](ancestors = 255703634)